### PR TITLE
WEB-45, WEB-46, WEB-47 more updates

### PR DESCRIPTION
**Pull request by wrooot**
Sorry about a high number of commits

### CODE REVIEW

**Comment by akrherz** [comment]
Some of these changes, I am not so sure about.

File: src/java/versions.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <versions>
     <!--<product name="wildfire" version="3.3.0" date="April 12, 2007" />-->
     <product name="openfire" version="4.0.2" date="March 21, 2016" />
-    <product name="openfire-beta" version="4.0.0.beta" date="December 13, 2015" />
+    <!--<product name="openfire-beta" version="4.0.0.beta" date="December 13, 2015" /> -->
```
Comment (akrherz) [inline src/java/versions.xml]: why is this necessary?  it is handy to have the template here

**Comment by wrooot** [comment]
A template? It shows up on the Beta page if not commented, i think. We are already on 4.0.2 final. So what is the point to have 4.0.0 Beta there?
